src/liquid/components/contact_type_property.html

```diff
@@ -5,13 +5,26 @@
       {{include.prop.friendly_name}}
     </label>
     <div class="control">
-      <input
-        name="{{ prop_name }}"
-        type="{% if include.prop.type == 'dob' %}date{% else %}text{% endif %}"
-        class="input"
-        {% if false and include.prop.type == 'regex' %} pattern="{{ include.prop.parameter }}" {% endif %}
-        {% if data[prop_name] %} value="{{ data[prop_name] }}" {% endif %}
-      />
+      {% if include.prop.type == 'select_one' or include.prop.type == 'select_multiple' %}
```

Comment:
there is a lot happening in this file. check out this change from Fred. https://github.com/medic/cht-user-management/pull/122/files#diff-cffbfbd30f96baa3bf1eecaf18f63d0fce51c820e87c0cf589510bd88bb2ce1eR14

Can we put this new stuff in a `contact_type_select` (or similar)?